spanner/opentelemetry_traces/Readme.md

```diff
@@ -0,0 +1,53 @@
+# Cloud Spanner OpenTelemetry Traces
+
+## Setup
+
+This sample requires [Java](https://www.java.com/en/download/) and [Maven](http://maven.apache.org/).
+
+1.  **Follow the set-up instructions in [the documentation](https://cloud.google.com/java/docs/setup).**
+
+2.  Enable APIs for your project.
+    
+    a. [Click here](https://console.cloud.google.com/flows/enableapi?apiid=spanner.googleapis.com&showconfirmation=true)
+    to visit Cloud Platform Console and enable the Google Cloud Spanner API.
+    
+    b. [Click here](https://console.cloud.google.com/flows/enableapi?apiid=cloudtrace.googleapis.com&showconfirmation=true)
+    to visit Cloud Platform Console and enable the Cloud Trace API.
+
+3.  Create a Cloud Spanner instance and database via the Cloud Plaform Console's
+    [Cloud Spanner section](http://console.cloud.google.com/spanner).
+
+4.  Enable application default credentials by running the command `gcloud auth application-default login`.
+
+## Run the Example
+
+1. Set up database configuration in the `OpenTelemetryUsage.java` class:
```

Comment:
nit: add a link to the actual file?

Reply:
Link can only be added once we merge. Will do that after this PR get merged